pkg/cluster/volumes.go

```diff
@@ -146,11 +146,11 @@ func (c *Cluster) resizeVolumes(newVolume spec.Volume, resizers []volumes.Volume
 }
 
 func (c *Cluster) volumesNeedResizing(newVolume spec.Volume) (bool, error) {
-	volumes, manifestSize, err := c.listVolumesWithManifestSize(newVolume)
+	vols, manifestSize, err := c.listVolumesWithManifestSize(newVolume)
```

Comment:
why rename volumes to vols? Doesn't seem to make the code clearer.

Reply:
because we have package named ``volumes``, and having the variable with the same name is not a good idea